AndelaOSP/art-android
# #156266491 Add security users toolbar

## What does this PR do?
Adds a security dashboard with a tool bar having a menu
## Description of Task to be completed?
- Add tests to check presence of a tool bar with a menu
- Create layouts for menu, toolbar and dashboard
- Add logic to render menu, toolbar and dashboard
## How should this be manually tested?
- Run the SecurityDashboardTest test file
- Launch the App on a device and login. You should be redirected to the security dashboard having a tool bar with a menu on it.
## What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/156266491
## Screenshot
Security dashboard showing the tool bar with a menu
<img width="358" alt="security_dashboard_with_toolbar" src="https://user-images.githubusercontent.com/29823251/37957716-53f9837a-31b7-11e8-9819-401b6384b0ba.png">
